central/graphql/resolvers/search.go

```diff
@@ -139,6 +139,11 @@ func (resolver *Resolver) getAutoCompleteSearchers() map[v1.SearchCategory]searc
 	}
 	if !features.PostgresDatastore.Enabled() {
 		searchers[v1.SearchCategory_VULNERABILITIES] = resolver.CVEDataStore
+	} else {
+		searchers[v1.SearchCategory_IMAGE_VULNERABILITIES] = resolver.ImageCVEDataStore
+		searchers[v1.SearchCategory_NODE_VULNERABILITIES] = resolver.NodeCVEDataStore
+		searchers[v1.SearchCategory_CLUSTER_VULNERABILITIES] = resolver.ClusterCVEDataStore
+		searchers[v1.SearchCategory_NODE_COMPONENTS] = resolver.NodeComponentDataStore
```

Comment:
Do we need to add a search for `SearchCategory_IMAGE_COMPONENTS` ?

Reply:
`ln137`